### PR TITLE
Improve issue templates

### PR DESCRIPTION
**Pull request by andreaslindeboom**
When creating an issue with a colleague, we noticed that the language in the issue creation/update templates could be slightly improved. This PR suggests some improvements.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/update_cheatsheet_proposal.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 ---
 
-Thanks you for proposing a update/refactor of a cheat sheet.
+Thank you for proposing to update/refactor one of our cheat sheets.
 
-Please provides the following information about your proposal:
+Please provide the following information about your proposal:
 
-- What update/refactoring do you want to perform? 
+1. What is missing or needs to be updated in the current iteration of the cheat sheet?
```
Comment (ThunderSon) [inline .github/ISSUE_TEMPLATE/update_cheatsheet_proposal.md]: Don't you feel like there should be an empty space in between to make even easier for the contributor and for the reviewer to separate the points?

Comment (andreaslindeboom) [reply .github/ISSUE_TEMPLATE/update_cheatsheet_proposal.md]: Thank you for your feedback 😃 

Just to clarify: do you mean separating the 'missing' or 'needs to be updated' aspects of the question? Or just adding newlines between the points?

Comment (ThunderSon) [reply .github/ISSUE_TEMPLATE/update_cheatsheet_proposal.md]: Newline between the 2 points. Apologies for the confusion, should have been clearer.

Comment (andreaslindeboom) [reply .github/ISSUE_TEMPLATE/update_cheatsheet_proposal.md]: Sorry for the delay, busy period. I see that you've merged it, thank you 😃

Comment (righettod) [reply .github/ISSUE_TEMPLATE/update_cheatsheet_proposal.md]: Thanks to you for your enhancement of the templates 😃

**Pull request merged**